src/erfa_common.jl

```diff
@@ -1,66 +1,41 @@
-
```

Comment:
Looks like these constants are used nowhere, perhaps are better suited in the newly-born `AstroBase.jl` package.  And `DPI` and `D2PI` are probably redundant, given we have `pi` in Julia.  Anyway, this is unrelated to this change.